src/app/public/modules/tabs/tabset.component.ts

```diff
@@ -132,6 +145,8 @@ export class SkyTabsetComponent
 
   private ngUnsubscribe = new Subject<void>();
 
+  private activeIndexOnLoad: number;
```

Comment:
Can we move `activeIndexOnLoad` above `ngUnsubscribe` so that they are in alphabetical order?

Reply:
Thanks!